git/operations.go

```diff
@@ -103,9 +103,13 @@ func addNote(workingDir, rev, notesRef string, note *Note) error {
 	return execGitCmd(workingDir, nil, nil, "notes", "--ref", notesRef, "add", "-m", string(b), rev)
 }
 
+// NB return values (*Note, nil), (nil, error), (nil, nil)
 func getNote(workingDir, notesRef, rev string) (*Note, error) {
 	out := &bytes.Buffer{}
 	if err := execGitCmd(workingDir, nil, out, "notes", "--ref", notesRef, "show", rev); err != nil {
+		if strings.Contains(err.Error(), "No note found for object") {
```

Comment:
This needs to match both `N` and `n` for different git versions. I'll make this change.